src/main/qss.js

```diff
@@ -275,6 +282,10 @@ gpii.app.qssWrapper.saveSettings = function (that, pspChannel, qssNotification,
     var saveButtonClickCount = pspChannel.model.saveButtonClickCount || 0;
     pspChannel.applier.change("saveButtonClickCount", saveButtonClickCount + 1, null, "PSP");
 
+    description = fluid.stringTemplate(description, {
```

Comment:
Probably not for this PR, but I think we should move the `description` message to the `qssWrapper`.